template.json

```diff
@@ -1,11 +1,11 @@
 {
   "package": {
     "dependencies": {
-      "@reduxjs/toolkit": "^1.1.0",
-      "@testing-library/jest-dom": "^4.2.4",
-      "@testing-library/react": "^9.3.2",
-      "@testing-library/user-event": "^7.1.2",
```

Comment:
I would recommend keeping `user-event` and using it as often as possible over `fireEvent` from `react-testing-library`. `user-event`'s goal is to eventually replace `fireEvent`, as `user-event` models actual user behavior in a browser more accurately rather than forcing you to manually fire the same events that a browser would.